x-pack/plugins/observability_ai_assistant/common/functions/visualize_esql.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FromSchema } from 'json-schema-to-ts';
+
+export const visualizeESQLFunction = {
+  name: 'visualize_query',
+  description:
+    'Use this function to visualize charts for ES|QL queries. DO NOT run the lens function after.',
```

Comment:
ℹ️ I had to instruct it to not run the lens suggestion after, not sure if this is the best way to do it but it worked